cminx/__init__.py

```diff
@@ -46,7 +46,7 @@
 from .rstwriter import RSTWriter
 
 
-def main(args: list[str]):
+def main(args: list[str] = sys.argv[1:]):
```

Comment:
Since you are using `argparse`, you shouldn't have to mess with `sys.argv` or other `sys` calls at all to get the arguments. If you remove this `args` parameter completely and just do `args = parser.parse_args()` on line 73, it should work. I tested it on my system and generated the CMaize documentation to make sure the change worked.

Reply:
Ah, there are tests that pass an argument list to main that I didn't know about. You probably need to keep the `main(args)` argument, then. Sorry about that.

Reply:
@ni-fgenois Ah, there are tests that pass an argument list to main that I didn't know about. You probably need to keep the `main(args)` argument, then. Sorry about that.